connector/src/main/java/org/geysermc/connector/utils/PluginMessageUtils.java

```diff
@@ -104,4 +111,8 @@ private static int getVarIntLength(int number) {
         }
         return 5;
     }
+
+    public static byte[] getEmoteRegisterData() {
+        return EMOTE_REGISTER_DATA;
+    }
```

Comment:
This would normally be replaced with `@Getter` but in this case, since its a static final make it public like the above.

Reply:
And `FLOODGATE_REGISTER_DATA`?  
Should I replace that too?

Reply:
I think this is fine the way it is.

Reply:
Just seen the rest of the file, disregard what I suggested.